test/e2e/page-objects/pages/test-dapp.ts

```diff
@@ -221,6 +233,10 @@ class TestDapp {
     });
   }
 
+  public async clickAddTokenToWallet() {
+    await this.driver.clickElement(this.addTokensToWalletButton);
```

Comment:
same as above `clickElementAndWaitForWindowToClose

Reply:
this one would not be needed as this action just opens a dialog but the dapp is still open

Reply:
oh it make sense! thank you :pray: